client/rpc/src/state/state_full.rs

```diff
@@ -21,6 +21,7 @@ use std::sync::Arc;
 use std::ops::Range;
 use futures::{future, StreamExt as _, TryStreamExt as _};
 use log::warn;
+use num_traits::ops::checked::CheckedSub;
```

Comment:
We should import that from `substrate` primitives instead of going directly to `num_traits`:
```
use sp_runtime::traits::CheckedSub
```